### ISSUE TITLE
couple of warnings during setup, possibly harmless

### ISSUE THREAD

**Issue by vesper8** [mayankk2308/purge-wrangler#29]
### System Configuration
Describe your system configuration here. State any anomalies (USB drives, etc.) involved as well.
- **Mac**: Macbook Pro late-2013
- **macOS Version**: 10.14.6
- **System GPUs**: Intel Iris Pro & NVIDIA GT 750M
- **eGPU**: Aorus Gaming Box + R9 Nano
- **Script Version**: v6 latest as of date of issue

### Issue Description
I just ran the setup and I can't say if everything installed fine yet, I'm about to reboot. I just wanted to paste my terminal log here first because I ran into 3 warnings.. not errors but warnings.. and perhaps you want to prevent these warnings as it can be alarming to see them even though they may be harmless

```
>> Setup eGPU

Plug-in eGPU. Press ESC if you are not plugging in eGPU.

External GPU	Radeon R9 FURY / NANO Series
GPU Arch 	Fiji
Thunderbolt 	2
Ti82 Enclosure 	No

Backing up...
Backup complete.

Downloading AMDLegacySupport...
purge-wrangler.sh: line 491: warning: command substitution: ignored null byte in input
Download complete.

Patching for AMD eGPUs...
sed: can't read : No such file or directory
Patches applied.

Detecting anomalies...

Detected System GPUs:
Intel Iris Pro
NVIDIA GeForce GT 750M

No anomalies expected.

Sanitizing system...
System sanitized.
purge-wrangler.sh: line 250: warning: command substitution: ignored null byte in input
Modifications complete.

Reboot to apply changes.

Reboot Now? [Y/N]:
```

**Comment by mayankk2308** [comment]
Thanks for the report. Could you uninstall and try again to see what you get?

**Comment by vesper8** [comment]
sure.. least I can do given all the work you've put into this : )

log of uninstall:

```
>> Uninstall

Uninstalling...
Reverting binaries...
sed: can't read : No such file or directory
Binaries reverted.
Sanitizing system...
System sanitized.
Uninstallation Complete.

Reboot to apply changes.
```

bash version:
```
✘ ✝ /  bash --version
GNU bash, version 5.0.7(1)-release (x86_64-apple-darwin18.5.0)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

I use zsh, which I installed manually. Not planning to upgrade to Catalina

I will paste the rest of the logs after I reboot, and reinstall, and reboot again

**Comment by vesper8** [comment]
reinstall log, same as earlier:

```
>> Setup eGPU

Plug-in eGPU. Press ESC if you are not plugging in eGPU.

External GPU	Radeon R9 FURY / NANO Series
GPU Arch 	Fiji
Thunderbolt 	2
Ti82 Enclosure 	No

Backing up...
Backup refreshed.

Downloading AMDLegacySupport...
/usr/local/bin/purge-wrangler: line 491: warning: command substitution: ignored null byte in input
Download complete.

Patching for AMD eGPUs...
sed: can't read : No such file or directory
Patches applied.

Detecting anomalies...

Detected System GPUs:
Intel Iris Pro
NVIDIA GeForce GT 750M

No anomalies expected.

Sanitizing system...
System sanitized.
/usr/local/bin/purge-wrangler: line 250: warning: command substitution: ignored null byte in input
Modifications complete.

Reboot to apply changes.

Reboot Now? [Y/N]:
```

**Comment by vesper8** [comment]
reinstall log, same as earlier:

```
>> Setup eGPU

Plug-in eGPU. Press ESC if you are not plugging in eGPU.

External GPU	Radeon R9 FURY / NANO Series
GPU Arch 	Fiji
Thunderbolt 	2
Ti82 Enclosure 	No

Backing up...
Backup refreshed.

Downloading AMDLegacySupport...
/usr/local/bin/purge-wrangler: line 491: warning: command substitution: ignored null byte in input
Download complete.

Patching for AMD eGPUs...
sed: can't read : No such file or directory
Patches applied.

Detecting anomalies...

Detected System GPUs:
Intel Iris Pro
NVIDIA GeForce GT 750M

No anomalies expected.

Sanitizing system...
System sanitized.
/usr/local/bin/purge-wrangler: line 250: warning: command substitution: ignored null byte in input
Modifications complete.

Reboot to apply changes.

Reboot Now? [Y/N]:
```

Status after reboot:
```
>> System Status

Ti82 Devices      Disabled
TB1/2 AMD eGPUs   Enabled
Legacy AMD eGPUs  Enabled
NVIDIA eGPUs      Disabled
Web Drivers       Absent
```

Also I tried Geekbench and Luxmark, as well as The Long Dark, and the eGPU is working great on internal monitor, no need for hdmi ghost adapter

**Comment by mayankk2308** [comment]
@vesper8 Appreciate the logs! So, I believe the warnings you see about null characters is not shown on the system-default bash 3.2. This warning was introduced in 4.4. Not a big problem and I should be able to resolve it in the next update (those file checks need a refresh anyway - not a good way to do it). I was a bit concerned about the **sed** error however, although I see now that the patches were applied successfully - error would indicate it should not have - so I was expecting to see TB1/2 AMD eGPUs to stay Disabled. I will investigate that further. Thanks again!

**Comment by mayankk2308** [comment]
@vesper8 This is now addressed in [6.0.2](https://github.com/mayankk2308/purge-wrangler/releases/tag/6.0.2).
